packages/server/src/utils/server-hooks.ts

```diff
@@ -1,4 +1,5 @@
 export const ServerHooks = {
+  Login: 'Login',
```

Comment:
Not sure about the name do you have a better Idea?

Reply:
I think this is fine for now, let's see if we can find a logic name scheme when more hooks will be added